### PR TITLE
fix(ga): bug 256 + ux fixes

### PR DESCRIPTION
**Pull request by unseen1980**
# Issue

https://github.com/SAP/guided-answers-extension/issues/256

## Description
Screenshot 2022-10-03 at 15 01 24

## Checklist for Pull Requests

- [ ] Supplied as many details as possible on this change
- [ ] Included the link to the associated issue in the Issue section above
- [ ] The code conforms to the general [development guidelines](https://github.com/SAP/guided-answers-extension/blob/main/docs/developer-guide.md).
- [ ] The code has **unit tests** where applicable and is easily unit-testable
- [ ] This branch is appropriately named for the associated issue

### CODE REVIEW

File: packages/webapp/src/webview/ui/components/Header/Filters/Filters.scss [hunk 1]
```diff
@@ -1,10 +1,20 @@
-.dialog-filter .ms-Dialog-content {
```
Comment (Klaus-Keller) [inline packages/webapp/src/webview/ui/components/Header/Filters/Filters.scss:1]: Why was this removed? Filter input now scrolls away.

**Review by Klaus-Keller — CHANGES_REQUESTED** [review]
@unseen1980, 
Tested locally and the issue is fixed, but while testing I recognized, that the search field on filter dialogs scrolls away:



This is not the case on the released version.

File: packages/webapp/src/webview/ui/components/Header/Filters/Filters.tsx [hunk 2]
```diff
@@ -210,50 +226,47 @@ export function Filters() {
...
+                        dialogContentProps={{ title: filterType[filter].title }}
+                        isOpen={filterType[filter].visibility}
+                        isBlocking={true}
+                        acceptButtonText={'Apply Filter'}
+                        cancelButtonText={'Cancel'}
+                        styles={{ main }}
+                        onAccept={() => filterType[filter].apply()}
+                        onCancel={resetFilter}
+                        onDismiss={resetFilter}>
+                        <UITextInput placeholder="Search!" value={query} onChange={searchFilter} />
```
Comment (Klaus-Keller) [inline packages/webapp/src/webview/ui/components/Header/Filters/Filters.tsx:262]: @unseen1980, was the change of the placeholder from **Search** to **Search!** intentionaly?

**Review by Klaus-Keller — COMMENTED** [review]
Tested again and the search input stays in place now. See my question about the placeholder text.

File: packages/webapp/src/webview/ui/components/Header/Filters/Filters.tsx (hunk 2 above)
Comment (unseen1980) [reply packages/webapp/src/webview/ui/components/Header/Filters/Filters.tsx]: Of course not, thanks for spotting it. I should spot working at night 😞

**Review by Klaus-Keller — APPROVED** [review]
- Code looks good to me
- Built and tested locally

Approved, thanks @unseen1980
